### DISCUSSION TITLE
Pagination of a nav collection

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#9219] [Troubleshooting]
I'm trying to paginate from a list of sub categories from my collection, however the pagination does not seem to work using the `{{ nav:collection:articles from="/articles/{segment_2}" as="articles" paginate="9" }}`

Pagination only shows when I use this `{{ collection:articles as="articles" paginate="9" }}` however that means I will have to keep tagging my pages as subcategories.

How can I use my pagination to show subcategories from the navigation tree?

**Comment by duncanmcclean** [comment]
Unfortunately, The `{{ nav }}` tag doesn't support pagination.
